packages/orbit-components/src/NotificationBadge/NotificationBadge.stories.tsx

```diff
@@ -186,3 +186,30 @@ Rtl.story = {
     info: "This is a preview of this component in RTL setup.",
   },
 };
+
+export const VisualTest = () => {
+  const Icon = getIcon(getIcons("Airplane"));
+  const types = Object.values(TYPE_OPTIONS);
+
+  return (
+    <div className="space-x-xs inline-flex">
```

Comment:
is this expected to be slightly lifted? Maybe better will be "flex items-center" instead of inline-flex?

Reply:
Ok, will add it after running the visual tests over tailwind, so that I don't get a false positive (because there will be a visual difference)

Reply:
Done ✅